packages/graphql-serve/src/components/serveHandler.ts

```diff
@@ -1,7 +1,8 @@
 import { buildGraphbackServer, GraphbackServer } from "../GraphbackServer";
+import { ConflictResolutionStrategyName } from "../runtime";
 
-export const serveHandler = async (argv: { model?: string, port?: number, datasync }): Promise<GraphbackServer> => {
-  const server = await buildGraphbackServer(argv.model, !!argv.datasync);
+export const serveHandler = async (argv: { model?: string, port?: number, datasync: boolean, conflict?: ConflictResolutionStrategyName, deltaTTL?: number }): Promise<GraphbackServer> => {
```

Comment:
The number of properties of `argv` is growing rather long. It might be time to create an interface for this?

```suggestion
export const serveHandler = async (args: GraphQLServeArgs): Promise<GraphbackServer> => {
```